zulipterminal/ui_tools/boxes.py

```diff
@@ -403,12 +408,51 @@ def stream_box_view(
         )
         self.stream_write_box.set_completer_delims("")
         self._setup_common_stream_compose(stream_id, caption, title)
-
         # Use and set a callback to set the stream marker
         self._set_stream_write_box_style(None, caption)
         urwid.connect_signal(
             self.stream_write_box, "change", self._set_stream_write_box_style
         )
+        self.check_stream_topic_edit_txt(self.check_stream_topic_edit_txt_flag)
+
+    @asynch
+    def check_stream_topic_edit_txt(
+        self, check_stream_topic_edit_txt_flag: bool
+    ) -> None:
+        narrow_key = primary_key_for_command("NARROW_MESSAGE_RECIPIENT")
+        narrow_markup = [
+            ("footer", "Help(?): "),
+            ("footer_contrast", f" {narrow_key} "),
+            ("footer", " Narrow to conversation."),
+        ]
+        send_key = primary_key_for_command("SEND_MESSAGE")
+        send_markup = [
+            ("footer", "Help(?): "),
+            ("footer_contrast", f" {send_key} "),
+            ("footer", " Send a message."),
+        ]
+        while check_stream_topic_edit_txt_flag:
```

Comment:
We don't want to use a busy loop for this, if we do take this approach. This sends the zulip-term CPU usage to 100%+.

An urwid on-change signal may be better, though note that while potentially helpful, `alt .` will not (or shouldn't) do anything if the recipients aren't valid, so we can avoid showing the notice in that case.